scripts/release.sh

```diff
@@ -133,6 +133,25 @@ git push "${REPOSITORY}" "${RELEASE_TAG}" ${GIT_DRYRUN:-}
 echo "Deploying release to maven central"
 mvn deploy -Prelease -DskipTests=true -DreleaseSigningKey="${GPG_KEY}" -DprocessAllModules=true ${MVN_DEPLOY_DRYRUN:-}
 
+echo "Creating binary distribution"
+mvn package -Pdist -DskipTests=true
+
+if ! command -v gh &> /dev/null
+then
+    echo "gh command could not be found. Please create a release by hand https://github.com/kroxylicious/kroxylicious/releases/new"
+else
+  echo "Creating GitHub release with binary distribution"
+  ORIGINAL_GH_DEFAULT_REPO=$(gh repo set-default -v | (grep -v 'no default repository' || true))
+  gh repo set-default $(git remote get-url ${REPOSITORY})
+  # create GitHub release via CLI https://cli.github.com/manual/gh_release_create
+  if [[ "${DRY_RUN:-false}" == true ]]; then
+    gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft
+    echo "Draft release created. This must be manually published."
+  else
+    gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md"
```

Comment:
What do you think of having the title follow the existing pattern of `Release 0.3.0` vs using the release tag like `v0.3.0`?

Reply:
I think it would add more complexity to the script (because I'd have to mess with the `$RELEASE_TAG` string to drop the "v").

Also IMHO it looks better and makes a bit more sense on the GitHub repository homepage if we omit the "Release" bit and stick with just the version number as the title, so it would read

> **Releases**
> v0.3.0

as opposed to the current

> **Releases**
> Release 0.3.0

Having "Release" in there twice feels a bit redundant to me.

Reply:
works for me, easier to correlate with the tag